sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -248,36 +250,26 @@ def batch_event_handler(event):
         HTTP response
 
     """
-    command = event["detail"]["container"]["command"]
-
-    # Get params from batch job command
-    instrument = command[1]
-    data_level = command[3]
-    descriptor = command[5]
-    start_date = datetime.strptime(command[7], "%Y%m%d")
-    version = command[9]
-
     # Get job status
     job_status = (
         models.Status.SUCCEEDED
         if event["detail"]["status"] == "SUCCEEDED"
         else models.Status.FAILED
     )
 
-    status_params = {
-        "status": job_status,
-        "instrument": instrument,
-        "data_level": data_level,
-        "descriptor": descriptor,
-        "start_date": start_date,
-        "version": version,
-        "job_definition": event["detail"]["jobDefinition"],
-        "job_log_stream_id": event["detail"]["container"]["logStreamName"],
-        "container_image": event["detail"]["container"]["image"],
-        "container_command": " ".join(command),
-    }
-
-    update_status_table(status_params)
+    # We injected our table ID into the job name
+    job_id = event["detail"]["jobName"].split("-")[-1]
```

Comment:
I first missed that we are adding table ID into the job name. Can you add comment in batch starter lambda that this is added which will used later to look up and update with batch information?

Reply:
Yes, I added a comment about this. Putting more information into the job name is what I think will help us avoid having to parse the command and get information out of it. If we can be explicit about our job names and add details there that should be easier than string splitting and messing with our CLI commands as well so our CLI doesn't need to be updated in the future, and we could just update our job naming convention.